openmetadata-ui/src/main/resources/ui/src/components/common/CronEditor/CronEditor.tsx

```diff
@@ -58,23 +66,24 @@ const getCron = (state) => {
       return getMonthCron(selectedMonthOption);
     case 'year':
       return getYearCron(selectedYearOption);
+    case 'hour':
     default:
-      return null;
+      return getHourCron(selectedHourOption);
```

Comment:
Why we are changing return value for default case

Reply:
@chirag-madlani , we don't need this will revert the change.

Reply:
Done.